client/src/Card.js

```diff
@@ -32,8 +32,9 @@ export default function Card() {
 }
 
 const QrStyled = styled.div`
```

Comment:
Hier vielleicht das 'r' von QR auch groß, weil du es in QRCode auch so hast? Wäre noch einen Tick einheitlicher